### PR TITLE
Orsi/sample apps

### PR DESCRIPTION
**Pull request by JasperHofman**
Pull request to merge Orsi Academy sample apps into Holohub public repository.

### CODE REVIEW

File: applications/orsi/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+endif()
+
+# Download datasets
+option(HOLOHUB_DOWNLOAD_DATASETS "Download datasets" ON)
+
+# Download ORSI sample data from NGC
+if(HOLOHUB_DOWNLOAD_DATASETS)
+  include(holoscan_download_data)
+  holoscan_download_data(orsi
+    #URL URL https://api.ngc.nvidia.com/v2/resources/nvidia/clara-holoscan/holoscan_orsi_academy_sample_data/versions/20231210/zip
```
Comment (jjomier) [inline applications/orsi/CMakeLists.txt:46]: URL needs to be changed once the datasets are public.

Comment (jjomier) [inline applications/orsi/CMakeLists.txt:46]:
```suggestion
```

File: applications/orsi/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    URL nvstaging/holoscan/holoscan_orsi_academy_sample_data:20231210
```
Comment (jjomier) [inline applications/orsi/CMakeLists.txt:47]:
```suggestion
    URL  https://api.ngc.nvidia.com/v2/resources/nvidia/clara-holoscan/holoscan_orsi_academy_sample_data/versions/20240206/zip
```

**Comment by jjomier** [comment]
@JasperHofman The datasets is now live, can you update the link?

File: operators/CMakeLists.txt [hunk 3]
```diff
@@ -28,9 +28,7 @@ add_holohub_operator(basic_network)
 add_holohub_operator(deltacast_videomaster DEPENDS EXTENSIONS deltacast_videomaster)
 add_holohub_operator(emergent_source DEPENDS EXTENSIONS emergent_source)
 add_holohub_operator(lstm_tensor_rt_inference DEPENDS EXTENSIONS lstm_tensor_rt_inference)
-add_holohub_operator(npp_filter)
-add_holohub_operator(prohawk_video_processing)
-add_holohub_operator(qt_video)
```
Comment (jjomier) [inline operators/CMakeLists.txt:33]: We should add these lines back

**Comment by JasperHofman** [comment]
@jjomier I added the lines back

**Pull request merged**